tests/simulated-unobstructed-airway_v0.0.0/ip900ap/README.md

```
@@ -0,0 +1,8 @@
+# Innovo IP900AP
+The Innovo IP900AP is a sensor designed to measure SPO2 (and heart rate).
+## Data Collection Procedure
+An AutoHotkey script was used to accept input from the user every two seconds which would then be saved to the EmotiBit as a user note through the EmotiBit oscilloscope. Therefore, there is only one file `EmotiBit.csv` which contains the IP900AP measured SPO2 levels every two seconds as user notes.
+## Files
+- `sit-stand-sit_v0.0.0_ip900ap.zip` contains `EmotiBit.csv`
+- `EmotiBit.csv` contains all the raw data recorded by the Emotibit
+  - Also includes the user notes which were used to save the SpO2 levels from the IP900AP
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)